src/subscriber.ts

```diff
@@ -763,6 +779,21 @@ export class Subscriber extends EventEmitter {
     const ackTimeSeconds = (Date.now() - message.received) / 1000;
     this.updateAckDeadline(ackTimeSeconds);
 
+    logs.ackNack.info(
+      'message (ID %s, ackID %s) ack',
+      message.id,
+      message.ackId,
+    );
+
+    if (ackTimeSeconds > this._99th) {
+      logs.slowAck.info(
+        'message (ID %s, ackID %s) ack took longer than the 99th percentile (%s s)',
```

Comment:
Since these are logs, we should probably be more explicit here. 99th percentile of what?